src/stories/TextFieldContainer.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styled from "styled-components";
+import TextFieldContainer from "../components/TextFieldContainer";
+
+export default {
+  title: "Text Field"
+};
+const Container = styled.div`
```

Comment:
Why do you have styling in your stories.js file? As far as I understood the styling only happens in the corresponding .js file

Reply:
If you want to demonstrate a use-case, it could make sense to have styling in a story file. 
An example:
You want to show how a text looks like in a full-width container and in a fixed-width container. Maybe the TextField has adds line-breaks. In this case, a story can have styling.

I am not sure about this story. What is the reason why you set the width to 400px?

Reply:
I wanted to text something, turned out it was useless so I deleted it :)